pandas/io/formats/style.py

```diff
@@ -254,14 +265,14 @@ def _translate(self):
         BLANK_CLASS = "blank"
         BLANK_VALUE = ""
 
-        def format_attr(pair):
+        def format_attr(pair: Dict[str, str]) -> str:
             return "{key}={value}".format(**pair)
 
         # for sparsifying a MultiIndex
         idx_lengths = _get_level_lengths(self.index)
         col_lengths = _get_level_lengths(self.columns, hidden_columns)
 
-        cell_context = dict()
+        cell_context: Dict = dict()
```

Comment:
subtypes

Reply:
mypy doesn't complain for what ever I put in the subtypes.

Anyway, I don't see ```cell_context``` being inserted with any values but in few places ```.get()``` gets called on it, I am very confused.

Reply:
Can you specify what the complaint is?

Reply:
No, I tried both with [stubgen](https://github.com/python/mypy/blob/master/mypy/stubgen.py) and [MonkeyType](https://github.com/Instagram/MonkeyType) and they didn't help. Also there is not really a good way to know if the types are actually good  or not because what no matter what I put in it mypy won't complain.

Plus, this annotation really doesn't add that much, so I dropped it.

Unless you have a different opinion about dropping that annotation.